pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/VarByteChunkForwardIndexReaderV4.java

```diff
@@ -330,14 +371,37 @@ private byte[] decompressAndRead(int docId)
       }
       return processChunkAndReadFirstValue(docId, offset, limit);
     }
+
+    private void initAndRecordRangesForDocId(int docId, List<ValueRange> ranges) {
+      // We'd practically end up reading entire metadata buffer
```

Comment:
expand the comment a bit, to explain why we'd read the entire metadata buffer

Reply:
Ack